### ISSUE TITLE
Behat tests won't run on PHP 7.2

### ISSUE THREAD

**Issue by anavarre** [acquia/blt#2662]
When running Behat tests on PHP 7.2 I get:

```
$ blt tests:behat:run
(snip)
  @javascript
  Scenario: Load a page with Javascript # features/Examples.feature:9
    Given I am on "/"                   # Drupal\DrupalExtension\Context\MinkContext::visit()
      Warning: count(): Parameter must be an array or an object that implements Countable in /var/www/html/blt/vendor/dmore/chrome-mink-driver/src/ChromePage.php line 35
(snip)

1 scenario (1 failed)
2 steps (1 failed, 1 skipped)
0m0.31s (21.79Mb)
[Testing\Behat]  Exit code 1 
[error]  Behat tests failed!
```

FYI a very similar PHP 7.2 error was fixed in core. See https://www.drupal.org/project/drupal/issues/2928846

When I switch to PHP 7.1 all is well.

```
$ blt tests:behat:run
(snip)
  @javascript
  Scenario: Load a page with Javascript # features/Examples.feature:9
    Given I am on "/"                   # Drupal\DrupalExtension\Context\MinkContext::visit()
    Then I should be on "/"             # Drupal\DrupalExtension\Context\MinkContext::assertPageAddress()
(snip)
2 scenarios (2 passed)
4 steps (4 passed)
0m2.18s (20.97Mb)

```

**Comment by grasmash** [comment]
Did you install composer dependencies using 7.2? This seems like a bug in dmore/chrome-mink-driver.

**Comment by anavarre** [comment]
> Did you install composer dependencies using 7.2?

Yes I did. I also tried to install them with PHP 7.1 and switch to 7.2 after to run Behat tests, but it doesn't make a difference and fails always on 7.2

I couldn't find any existing issue in https://gitlab.com/DMore/chrome-mink-driver/issues but I'm surprised none would have been reported against PHP 7.2 yet.

**Comment by malikkotob** [comment]
This is indeed an upstream issue with chrome-mink-driver. They actually already have a fix for it that is available as of `2.5.0`. BLT is currently pinning to `2.4.0`, because of an issue that was encountered in #2138. That issue seems to have been since fixed (https://github.com/acquia/blt/issues/2138#issuecomment-341283635), so #2674 aims to undo the pinning by no longer explicitly requiring chrome-mink-driver.

**Comment by anavarre** [comment]
If I unpin `dmore/chrome-mink-driver`, I get version `2.6.2` and tests start working again on PHP 7.2
